examples/basic/components/link/props.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+  text: { description: 'link text' },
+  href: {
+    description: 'link href',
+  },
+}
+
+export type { Props } from './index'
```

Comment:
an example of just using Props from the component itself, rather than the contrived example in Button, where the Props are declared there :)